File: modules/spring-web/testdata/java/providers/linemarkers/UserHandler.java
import org.springframework.web.reactive.function.server.ServerRequest;
import org.springframework.web.reactive.function.server.ServerResponse;
import reactor.core.publisher.Mono;

@org.springframework.stereotype.Component
public class UserHandler {
    public Mono<ServerResponse> getUser(ServerRequest request) {
        return Mono.empty();
    }
    public Mono<ServerResponse> getUserCustomers(ServerRequest request) {
        return Mono.empty();
    }
    public Mono<ServerResponse> deleteUser(ServerRequest request) {
        return Mono.empty();
    }
}

File: modules/spring-web/testdata/kotlin/providers/linemarkers/UserHandler.kt
import org.springframework.stereotype.Service
import com.explyt.apispring.Response
import com.explyt.apispring.config.properties.RewardConfiguration
import com.explyt.apispring.config.properties.TimingConfiguration
import com.explyt.apispring.data.*
import org.springframework.stereotype.Service
import org.springframework.web.reactive.function.server.ServerRequest
import org.springframework.web.reactive.function.server.ServerResponse

data class User(val id: Int) : com.explyt.apispring.rest.response.Response {}

@Service
class UserHandler() {
    suspend fun createUser(request: ServerRequest): ServerResponse {
        return Response.ok(User(1))
    }

    suspend fun getUserById(request: ServerRequest): ServerResponse {
        return Response.ok(User(1))
    }

    suspend fun updateUserData(request: ServerRequest): ServerResponse {
        return Response.ok(User(1))
    }
}